daffodil-lib/src/main/scala/org/apache/daffodil/cookers/EntityReplacer.scala

```diff
@@ -44,7 +44,7 @@ class EntitySyntaxException(msg: String) extends Exception(msg)
 final class EntityReplacer {
 
   val dfdlEntityName = "NUL|SOH|STX|ETX|EOT|ENQ|ACK|BEL|BS|HT|LF|VT|FF|CR|SO|SI|DLE|DC[1-4]|NAK|SYN|ETB|CAN|EM|SUB|ESC|FS|GS|RS|US|SP|DEL|NBSP|NEL|LS"
-  val dfdlCharClassEntityName = "NL|WSP|WSP\\*|WSP\\+|ES"
+  val dfdlCharClassEntityName = "NL|WSP|WSP\\*|WSP\\+|ES|LSP|LSP\\*|LSP\\+|SP\\*|SP\\+"
```

Comment:
Mentioned elsewhere, but I wonder if we want to remove SP from `dfdlEntityName` and `entityCharacterUnicode` and add it to `dfdlCharClassEntityName`. It's technically not a char class, but I think maybe all the logic just works if we treat it as a character class of just a single character. That way SP is treated just like WSP and LS as far as all the reduce delim buf logic and delimiter logic goes.